Formula/watson.rb

```diff
@@ -42,6 +44,20 @@ class Watson < Formula
 
   def install
     virtualenv_install_with_resources
+
+    if build.with? "completion"
```

Comment:
I think we can go ahead and install completion by default, since that's what most people will expect.